src/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -154,6 +154,7 @@ public void SendFile_Synchronous(IPAddress listenAt, bool sendPreAndPostBuffers,
                 client.SendFile(filename, preBuffer, postBuffer, TransmitFileOptions.UseDefaultWorkerThread);
 
                 client.LingerState = new LingerOption(true, LingerTime);
```

Comment:
>I think here use Shutdown() is better, because the number of LingerTime is arbitrary.

Then if Shutdown is better, shouldn't the client.LingerState line be removed? It seems duplicative now if you are using Shutdown.